packages/graphql/apollo/src/services/ApolloService.ts

```
@@ -70,7 +70,7 @@ export class ApolloService {
           switch (this.platformName) {
             case "express":
               // use newer graphql express middleware
-              const expressVersion = await import("express/package.json").then((pkg) => pkg.default.version);
+              const expressVersion = await import("express/package.json", { with: { type: "json" } }).then((pkg) => pkg.default.version);
```

Biome (2.3.14)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)